lib/shiba/parsers/mysql_select_fields.rb

```diff
@@ -66,6 +67,18 @@ def parse_fields
 
           sc.scan(/,/)
         end
+
+        # resolve table aliases
+        if sc.scan(/ `.*?`\.`(.*?)` `(.*?)`/)
```

Comment:
this syntax can also be `table AS alias`, let's support that too

Reply:
other stuff:

- tables can be not quoted
- only handles a single table

If we're gonna bother, we should go a little further...

Reply:
This parser only parses output from show warnings, which only has this format (e.g. no as, always quoted)

Reply:
roger